vhdbuilder/scripts/linux/mariner/tool_installs_mariner.sh

```diff
@@ -10,6 +10,8 @@ installBcc() {
     echo "Installing BCC tools..."
     dnf_makecache || exit $ERR_APT_UPDATE_TIMEOUT
     dnf_install 120 5 25 bcc-tools || exit $ERR_BCC_INSTALL_TIMEOUT
+    echo "Installing BCC examples..."
```

Comment:
If we expect more bcc packages to be added sometime in the future, it could be good to lay the groundwork for that now. 

https://github.com/Azure/AgentBaker/blob/ff3ff212947242de5ee8ca7d8d87a3d1d8d84dbc/parts/linux/cloud-init/artifacts/mariner/cse_install_mariner.sh#L20

Uses a For loop to make adding additional packages to the "install list" very easy. We could use that same logic here.

However, if we don't foresee additional packages ever being added to this function call, I think it looks great as is and you can resolve this comment :)

Reply:
Great point Sean! It is unlikely that there are other bcc packages. In the future new tools and more examples may be added into the existing packages, so I will keep the change as is here.